src/appMain/sdl_preloaded_pt.json

```diff
@@ -2291,6 +2387,15 @@
                 "priority": "NONE",
                 "default_hmi": "NONE",
                 "groups": ["BaseBeforeDataConsent"]
+            },
+            "pre_consent_passengersRC": {

```

Comment:
@yang1070 
EXTERNAL_PROPRIETARY option adds "data/device consent" to regular policy permissions. This is necessary for RC apps to fit this concept.